aspnetcore/fundamentals/index/snapshots/2.x/Startup1.cs

```diff
@@ -0,0 +1,18 @@
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddMvc().SetCompatibilityVersion
```

Comment:
Wrap the entire `SetCompatibilityVersion` method call to the next line. I think it looks nicer if its parameter is on the same line.